repository/src/main/java/org/alfresco/transform/client/registry/CombinedConfig.java

```diff
@@ -113,9 +74,7 @@ public CombinedConfig(Log log)
             protected void readJson(JsonNode jsonNode, String readFrom, String baseUrl)
             {
                 TransformConfig transformConfig = jsonObjectMapper.convertValue(jsonNode, TransformConfig.class);
-                transformConfig.getTransformOptions().forEach((key, map) -> combinedTransformOptions.put(key, map));
-                transformConfig.getTransformers().forEach(transformer -> combinedTransformers.add(
-                        new TransformAndItsOrigin(transformer, baseUrl, readFrom)));
+                addTransformConfig(transformConfig, readFrom, baseUrl);
```

Comment:
addTransformConfig(TransformConfig) is a new method to make it easier for the t-router to use the common CombinedTransformConfig class in the model.